security/pkg/k8s/controller/workloadsecret_test.go

```diff
@@ -163,7 +163,7 @@ func TestSecretController(t *testing.T) {
 		}
 		controller, err := NewSecretController(createFakeCA(), defaultTTL,
 			tc.gracePeriodRatio, defaultMinGracePeriod, false, client.CoreV1(), false,
-			metav1.NamespaceAll, webhooks)
+			[]string{metav1.NamespaceAll}, webhooks)
```

Comment:
Please test multiple values as well. Same for the following.

Reply:
@myidpt Thanks for your comments. I just checked the current test cases, it seems they do not depend on the listened namespace, because they are just calling the controller.xxx and then compare the real actions with expected actions. they are always same. Do you think it is necessary to add multiple values testing here? if so, i can image 2 solutions:
1. add namespaces checking in each controller `AddFunc/DeleteFunc/UpdateFunc`. if the current action occurs in listened namespaces, then continue. otherwise, just return.
2. refactor `NewSecretController` method to allow pass `ListerWatcher`, then we can use `fcache.NewFakeControllerSource()` for testing. but I think it should belong to part of `MultiNamespaceListerWatcher` testing.

What do you think?

Reply:
I see, thanks for the explanation. I think it's fine to have the multi-namespace testing in the MultiNamespaceListerWatcher tests.

Reply:
@myidpt Thanks Oliver.